services-core/gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=4.8.0-SNAPSHOT
+VERSION_NAME=4.9.0-SNAPSHOT
```

Comment:
Just a note that since this version is always the same as the one in the root `gradle.properties`, it's probably safe to delete it.